### PR TITLE
Update edit for orders and products controller of admin page

### PR DESCRIPTION
**Pull request by thangdangminh**
Add function accept request product from admin's side

### CODE REVIEW

**Comment by thangdangminh** [comment]
ready

File: app/Http/Controllers/Admin/ProductController.php [hunk 1]
```diff
@@ -24,4 +26,76 @@ public function index()
...
+                'description' => $requestProduct->description,
+            ]);
+        }
+
+        return redirect()->route('products.index');
+    }
+
+    public function requestProductShow($id)
+    {
+        $request = RequestProduct::findOrFail($id);
```
Comment (tranducquoc) [inline app/Http/Controllers/Admin/ProductController.php]: xử lý không tìm thấy

File: resources/views/admin/products/create.blade.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                        'class' => 'custom-file-input',
+                        'id' => 'image',
+                    ]) !!}
+                    {!! Form::label('image', @trans('admin.table.product.image'), [
+                        'class' => 'custom-file-label',
+                    ]) !!}
+                </div>
+            </div>
+            {!! Form::submit(@trans('common.form.button.submit'), ['class' => 'btn btn-primary']) !!}
+            <button type="reset" class="btn btn-danger">Reset</button>
```
Comment (tranducquoc) [inline resources/views/admin/products/create.blade.php]: i18n

**Comment by thangdangminh** [comment]
ready

**Pull request merged**